src/init.cpp

```diff
@@ -928,12 +928,15 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
         return InitError(_("Prune cannot be configured with a negative value."));
     }
     nPruneTarget = (uint64_t) nSignedPruneTarget;
-    if (nPruneTarget) {
+    if (nPruneTarget / 1024 / 1024 == 1) {  // manual pruning: -prune=1
```

Comment:
Can we please store and compare the option value before multiplication? Doing a division here, though it achieves the correct effect, seems circuitous and unclear.